README.md

```diff
@@ -93,8 +93,7 @@ We provide an extensive tutorial in the [notebooks](notebooks) folder with guide
 
 ## Installation
 
-Mab2Rec can be installed from PyPI using ``pip install mab2rec`` or by building from source by following the instructions
-in our [documentation](https://fidelity.github.io/mab2rec/installation.html).
+Mab2Rec requires **Python 3.7+** and can be installed from PyPI using ``pip install mab2rec`` or by building from source as shown in [installation instructions](https://fidelity.github.io/mab2rec/installation.html).
```

Comment:
I added Python 3.7+ dependency at top level README. Many people tripped over this based on User Group chat. The Python dependency is only written in the doc pages under installation. I doubt many go there, and just install the reqs.txt without keeping an eye Python version. This should help it, hopefully. I did the same in all other libs as well.